src/godirect.js

```diff
@@ -9,12 +10,12 @@ const godirect = {
   * @param {config} config
   * @returns {Promise} Promise object represents a Device instance
   */
-  async createDevice(bleDevice, { open = true, startMeasurements = true } = {}) {
-    let adapter = bleDevice;
+  async createDevice(baseDevice, { open = true, startMeasurements = true, bluetooth = true } = {}) {
+    let adapter = baseDevice;
 
     // If not a go direct adapter, assume a web bluetooth device
```

Comment:
Can you update this comment. It should say something like: `// If not a godirect adapter, assume web device`